var/spack/repos/builtin/packages/scale/package.py

```diff
@@ -0,0 +1,84 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import os
+
+from spack import *
+
+
+class Scale(MakefilePackage):
+    """SCALE (Scalable Computing for Advanced Library and Environment) is
+    a basic library for weather and climate model of the earth and planets
+    aimed to be widely used in various models.
+    The SCALE library is developed with co-design by researchers of
+    computational science and computer science."""
+    homepage = "https://scale.riken.jp/"
+    url      = "https://scale.riken.jp/archives/scale-5.4.4.tar.gz"
+
+    maintainers = ['t-yamaura']
+
+    version('5.4.4', sha256='7d0ec4069c15d8b9ec7166f32c9a2eda772d975a8e08e420e6b16891ceebb316', preferred=True)
+    version('5.3.6', sha256='3ab0d42cdb16eee568c65b880899e861e464e92088ceb525066c726f31d04848')
+    version('5.2.6', sha256='e63141d05810e3f41fc89c9eb15e2319d753832adabdac8f7c8dd7acc0f5f8ed')
+
+    depends_on('openmpi', type=('build', 'link', 'run'))
+    depends_on('netcdf-c')
+    depends_on('netcdf-fortran')
+    depends_on('parallel-netcdf')
+
+    patch('fj-own_compiler.patch', when='%fj')
+
+    parallel = False
+
+    def setup_build_environment(self, build_env):
+        build_env.set('PREFIX', self.prefix)
+
+    def build(self, spec, prefix):
+        scale_sys_str = ''
+        if self.spec.satisfies('platform=linux %gcc'):
+            scale_sys_str = 'Linux64-gnu-ompi'
```

Comment:
```suggestion
        if self.spec.satisfies('platform=linux %gcc ^openmpi'):
            scale_sys_str = 'Linux64-gnu-ompi'
```
If Makedefs.* include openmpi specific configuration, we need to ladd dependency condition.